### DISCUSSION TITLE
About verifying my email

### DISCUSSION THREAD

**Discussion by Ogannesson** [community/community#23137] [New to GitHub]
<p>I added a second email to my account, it’s an educational email provided by my university, but I can’t receive the verification code that github sends me</p>

**Comment by jfullstackdev** [comment]
<p>even without verifying it, all commits linked to that will automatically be added to your GitHub contribs but if you really need to verify it, make sure it’s really not sending and try this one</p>
<p><a href="https://support.github.com/contact/account">https://support.github.com/contact/account</a></p>

**Comment by Ogannesson** [comment, marked as answer]
<p>In order to apply for github’s student package, I have to verify my education email. After contacting github support, it was determined that the verification email was rejected by my email service provider. After contacting the email admin, I have successfully verified it.</p>

**Reply by danishmustafa86** [reply]
I have the same issue.

**Comment by queenofcorgis** [comment]
<p>Happy to hear you got this resolved <a class="mention" href="/unbiseptium">@Unbiseptium</a> - I’ve marked this as the solution :tada:</p>

**Comment by MAHENDRA-BUOY** [comment]
The webpage at might be temporarily down or it may have moved permanently to a new web address. 
ERR_ADDRESS_INVALID HELP MEE

**Reply by pokiopok** [reply]
I am having the same issue, have already contacted my university and they say the problem is not at their end

**Comment by nataliavramidou** [comment]
how to contact the email admin?
